### PR TITLE
[DOCFIX] Update security docs

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

**Comment by gpang** [comment]
@ZacBlanco

File: docs/en/operation/Security.md [hunk 1]
```diff
@@ -300,15 +308,18 @@ client.
 The most common impersonation error applications may see is something like
 `User yarn is not configured for any impersonation. impersonationUser: foo`. This is most likely
 due to the fact that the Alluxio servers have not been configured to enable impersonation for that
-user. To fix this, the Alluxio servers must be configured to enable impersonation for the user in question (yarn in the example error message).
+user. To fix this, the Alluxio servers must be configured to enable impersonation for the user
```
Comment (ZacBlanco) [inline docs/en/operation/Security.md]: nit: Can you put this sentence on a new line?

Comment (gpang) [reply docs/en/operation/Security.md]: updated

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by gpang** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Pull request merged**